test/e2e/NoRegression.spec.ts

```diff
@@ -0,0 +1,216 @@
+import fc from '../../src/fast-check';
+
+const testFunc = (value: unknown) => {
+  const repr = fc.stringify(value);
+  for (let idx = 1; idx < repr.length; ++idx) {
+    if (repr[idx - 1] === repr[idx] && repr[idx] !== '"') {
+      return false;
+    }
+  }
+  return true;
+};
+
+describe(`NoRegression`, () => {
+  // Bumping from one patch of fast-check to another is not supposed
+  // to change the values that will be generated by the framework.
+  //
+  // Except in case of a real bug causing the arbitrary to be totally unusable.
+  //
+  // This suite checks this invariant stays true.
+  // Moreover, the framework should build consistent values throughout all the versions of node.
+  const settings = { seed: 42, verbose: 2 };
+
+  it('float', () => {
+    expect(() => fc.assert(fc.property(fc.float(), v => testFunc(v)), settings)).toThrowErrorMatchingSnapshot();
+  });
+  it('double', () => {
+    expect(() => fc.assert(fc.property(fc.double(), v => testFunc(v)), settings)).toThrowErrorMatchingSnapshot();
+  });
+  it('integer', () => {
+    expect(() => fc.assert(fc.property(fc.integer(), v => testFunc(v)), settings)).toThrowErrorMatchingSnapshot();
+  });
+  it('nat', () => {
+    expect(() => fc.assert(fc.property(fc.nat(), v => testFunc(v)), settings)).toThrowErrorMatchingSnapshot();
+  });
+  it('maxSafeInteger', () => {
+    expect(() =>
+      fc.assert(fc.property(fc.maxSafeInteger(), v => testFunc(v)), settings)
+    ).toThrowErrorMatchingSnapshot();
+  });
+  it('maxSafeNat', () => {
+    expect(() => fc.assert(fc.property(fc.maxSafeNat(), v => testFunc(v)), settings)).toThrowErrorMatchingSnapshot();
+  });
+  it('string', () => {
+    expect(() => fc.assert(fc.property(fc.string(), v => testFunc(v)), settings)).toThrowErrorMatchingSnapshot();
+  });
+  it('asciiString', () => {
+    expect(() => fc.assert(fc.property(fc.asciiString(), v => testFunc(v)), settings)).toThrowErrorMatchingSnapshot();
+  });
+  // // Jest Snapshot seems not to support incomplete surrogate pair correctly

```

Comment:
Waiting updates on https://github.com/facebook/jest/issues/8403 in order to state whether or not this test should be kept (if considered to be a bug).